### DISCUSSION TITLE
Maybe use PureImage

### DISCUSSION THREAD

**Discussion by joh-klein** [bostrom/text-to-image#174] [Ideas]
Hi, just a thought: maybe you could use [PureImage](https://www.npmjs.com/package/pureimage) to get rid of the node-canvas dependency?

**Comment by bostrom** [comment]
Hi @joh-klein, thanks for the suggestion. It looks promising. It seems to be in early development still with some missing features like bold text, so let's see how compatible it is. But as a future direction I like it.
